travispy/entities/log.py

```diff
@@ -1,23 +1,38 @@
 from ._entity import Entity
 
+from travispy._helpers import get_archived_log
+
 
 class Log(Entity):
     '''
     :ivar int job_id:
         Jod ID.
 
-    :ivar str body:
-        Log body.
-
     :ivar str type:
     '''
 
     __slots__ = [
         'job_id',
-        'body',
+        '_body',
         'type',
     ]
 
+    def __init__(self, session):
```

Comment:
'body' exists in the payload.  If it was a recent job, the 'body' is the log text.  After a while the job is archived and body becomes ''.

If you care about backwards compatibility, the 'body' attribute must always contain the log text.
If you do not care about backwards compatibility, are you going to push the release number up to 1.0 per semver?